src/models/Clock.java

```diff
@@ -1,18 +1,61 @@
 package models;
 
+import logic.TransitionSystem;
+
 import java.util.Objects;
 
 public class Clock extends UniquelyNamed {
-    public Clock(String name, String ownerName) {
+    /**
+     * If true then this clock is used in the scope of multiple {@link TransitionSystem} or {@link Automaton}.
+     * As an example, quotients creates a new "quo_new" clock unless one already exists.
+     * Instead of creating a new and different clock the same clock is reused.
+     */
+    private final boolean isGlobal;
+
+    /**
+     * Constructs a new clock.
+     *
+     * @param name The original name and initial unique name of the clock.
+     * @param ownerName The owner of the clock. E.g. the name of the {@link Automaton}.
```

Comment:
I don't know why we use the name of the owner and not a reference. However, if a reference was stored, then it can prolong its lifetime of it. I am not entirely sure if that would be expected. I think this way of handling ownership should be redesigned in the future.

Reply:
Yes, it's definitely something for the future.

Reply:
I created #97 for this.